server/src/lib/dokku/git/sync.ts

```diff
@@ -0,0 +1,15 @@
+import { NodeSSH, SSHExecOptions } from 'node-ssh';
+
+export const sync = async (
+  ssh: NodeSSH,
+  appName: string,
```

Comment:
can we use named arguments here? It's fairly easy to do a mistake when there are a lot of args otherwise

Reply:
Sure, will add.